src/backend/commands/policy.c

```diff
@@ -391,6 +391,12 @@ RemovePolicyById(Oid policy_id)
 
 	systable_endscan(sscan);
 
+	/*
+	 * CBDB GITHUB ISSUE:
+	 * https://github.com/cloudberrydb/cloudberrydb/issues/504
+	 */
+	MetaTrackDropObject(PolicyRelationId, policy_id);
```

Comment:
Tracking object looks only happen on QD. Shall we follow this convention? There's no wrong to run this function on QEs, but we'd better define the behavior clearly in a whole.

Reply:
Good catch.